Explorer/Assets/Scripts/ECS/Unity/Materials/Systems/CreateMaterialSystemBase.cs

```diff
@@ -34,10 +36,14 @@ protected bool TryGetTextureResult(ref Promise? promise, out StreamableLoadingRe
             return promise.Value.TryGetResult(World, out textureResult);
         }
 
-        protected static void TrySetTexture(Material material, ref StreamableLoadingResult<Texture2D> textureResult, int propId)
+        protected static void TrySetTexture(Material material, ref StreamableLoadingResult<Texture2D> textureResult, int propId, TextureComponent? textureComponent)
```

Comment:
Pass `textureComponent` by `in`, otherwise it may create a `memcpy` overhead especially when there are a lot of materials to process per frame